modules/rosa-upgrading-automatic-ocm.adoc

```diff
@@ -1,16 +1,30 @@
 // Module included in the following assemblies:
 //
 // * rosa_upgrading/rosa-upgrading.adoc
+// * rosa_upgrading/rosa-upgrading-sts.adoc
+
+ifeval::["{context}" == "rosa-upgrading-sts"]
+:sts:
+endif::[]
 
 [id="rosa-scheduling-upgrade_{context}"]
 = Scheduling automatic upgrades
 
+You can schedule automatic upgrades for a {product-title} cluster 
+ifdef::sts[]
+that uses the AWS Security Token Service (STS) 
+endif::sts[]
+through the OpenShift Cluster Manager (OCM) console.
+
+ifdef::sts[]
+.Prerequisites
 
-There are three methods that can be used to upgrade {product-title} clusters. This procedure describes scheduling automatic upgrades.
+* If you are upgrading your cluster from 4.7 to 4.8, you have upgraded the AWS Identity and Access Management (IAM) account-wide roles and policies to version 4.8. You have also updated the `cloudcredential.openshift.io/upgradeable-to` annotation in the `CloudCredential` custom resource. For more information, see _Prerequisites for upgrading from 4.7 to 4.8_.
```

Comment:
nit to apply or ignore across three files: "you are upgrading" is eye catching.  Is it any more direct as "You upgrade from 4.7 to 4.8, ..." or "For a 4.7 to 4.8 upgrade,..."?

If you revised the heading from "Prereqs" to "Preparing", then friendly reminder to update here.

Reply:
Thanks Mike for the suggestion. I am going to opt to leave the prerequisite as is is, because the condition is clear and explicit.

Thanks also for the reminder to update the reference.